subprojects/docs/src/docs/release/notes.md

```diff
@@ -312,6 +312,7 @@ This is not the case anymore as the `source` filed is now declared as `private`.
 - Removed `JavaBasePlugin.configureForSourceSet`
 - Removed `JDepend.classesDir`
 - Removed `IdeaPlugin.performPostEvaluationActions` and `EclipsePlugin.performPostEvaluationActions`
+- Removed `ConfigurableReport.setDestination(Object)`
```

Comment:
What should be the style now in the release notes? I am seeing `Removed <property|method> from <class>` and `Removed <class.property|method>`. I think we should be consistent, right?

Reply:
Agreed. Let's make these consistent when we're done with removing deprecated code. Otherwise, I'm afraid we'd end up with lots of merge conflicts.